style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Poppins;
+  src: url("./resources/font/Poppins-Regular.ttf");
+}
+
+@font-face {
+  font-family: Poppins-Bold;
+  src: url("./resources/font/Poppins-Bold.ttf");
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    margin: 0;
+  }
+}
+
+html {
+  background: #e5e5e5;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-direction: row;
+  background: white;
+  width: 100vw;
+}
+
+.menu {
+  float: right;
+  padding-left: 60%;
+  padding-top: 2%;
+}
+
+.headline-container {
+  height: 60vh;
+  width: 100vw;
+}
+
+article {
+  display: flex;
+  min-height: 113vh;
+  flex-direction: column;
+  background-color: white;
+  border-bottom-left-radius: 15%;
+  background-image: url("./resources/images/header-shapes\ mobilebackground-Main.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: 99% -23%;
```

Comment:
you did well adding the background image, kindly use the right spacing between the nav and the background image for your project as it is seen in the Figma template given 😄 

Figma design :
![image](https://github.com/Afsheen96/Portfolio/assets/20537712/f7cb6029-3532-40db-b212-3822a4ead1eb)

Your design :
![image](https://github.com/Afsheen96/Portfolio/assets/20537712/7b1888ed-de53-4921-9605-dd1428405007)